src/services/schemas/properties/SchemaPropertyService.ts

```diff
@@ -82,13 +82,11 @@ export abstract class SchemaPropertyService {
   }
 
   public mapRequestValue(value: SchemaValue): SchemaValue | undefined {
-    const mappedValue = this.request(value)
-
-    if (this.isDefaultValue(mappedValue)) {
+    if (this.isDefaultValue(value)) {
       return undefined
     }
 
-    return mappedValue
+    return this.request(value)
```

Comment:
This is where we were comparing a mapped value with an unmapped value. Most of the time that was fine because we were stringifying to compare. But comparing two unmapped values produces a more accurate comparison.